core-web/package.json

```diff
@@ -115,7 +115,7 @@
         "node-fetch": "^2.6.1",
         "primeflex": "3.3.1",
         "primeicons": "7.0.0",
-        "primeng": "17.18.11",
+        "primeng": "17.18.15",
```

Comment:
Did you check the storybook to double-check if the other controls are not broken?